src/lib.rs

```diff
@@ -13,8 +13,8 @@
 //! `Branch` and `BranchMut`, types for representing branches in tree-formed
 //! data as well as methods of search.
 
-#![no_std]
-#![deny(missing_docs)]
+#![cfg_attr(not(feature = "persistance"), no_std)]
+#![warn(missing_docs)]
```

Comment:
Why the change from `deny` to `warn`? I assume it's a left over did during the development, it should be change back.

Reply:
fixed